app/main.py

```diff
@@ -1 +1,19 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> str:
+    cmd, current_file_name, new_path = command.split()
```

Comment:
You need to validate the command:
Your code will fail if the command will be one of:
mv file1.txt
mv file1.txt file2.txt file3.txt

Reply:
Rename variables also:
current_file_name -> input_file_path or source_file_path
new_path -> output_file_path or destination_file_path